pkg/mount-manager/fake-safe-mounter.go

```diff
@@ -38,10 +38,38 @@ func NewFakeSafeMounter() *mount.SafeFormatAndMount {
 			return []byte(script.output), script.err
 		*/
 	}
-	fakeMounter := &mount.FakeMounter{MountPoints: []mount.MountPoint{}, Log: []mount.FakeAction{}}
-	fakeExec := mount.NewFakeExec(execCallback)
+
+	fakeMounter = &mount.FakeMounter{MountPoints: []mount.MountPoint{}, Log: []mount.FakeAction{}}
+	fakeExec    = mount.NewFakeExec(execCallback)
+)
+
+func NewFakeSafeMounter() *mount.SafeFormatAndMount {
 	return &mount.SafeFormatAndMount{
 		Interface: fakeMounter,
 		Exec:      fakeExec,
 	}
 }
+
+type FakeBlockingMounter struct {
+	*mount.FakeMounter
+	OkToRun   chan struct{}
+	OpStarted chan struct{}
+}
+
+func (mounter *FakeBlockingMounter) Mount(source string, target string, fstype string, options []string) error {
+	mounter.OpStarted <- struct{}{}
```

Comment:
nit: `mountOpStarted`? I think the mounter also does the unmount (maybe in 1.14+ only) so we should disambiguate